### PR TITLE
feat(#742): add translations for error pages

### PR DESCRIPTION
**Pull request by salihdev0**
# Description

added translations for error pages

**discord username: ss.ibrahimbas#4990 **

**closes #issue**
#742

Please describe your changes. Also describe your aim and content. Do not forget to list the dependencies required caused by those changes.

Added translations for 404 page.
Added translations for the _error page.

## Things to check before creating a PR
- [x] I have inspected my topic, checked.
- [ ] If it is a core feature, executed detailed tests.

## Creating PR rules
- [x] PR must be created for an issue with approved tag. Otherwise PR will be rejected.
- [x] Relevant issue number: The issue number related to PR must be attached to head of PR header, after prefix after prefix # must be attached in parenthesis. A header like this could be used "prefix(#issue_number): PR header" .
- [x] A descriptive and understandable title: The PR title should clearly describe the nature and purpose of the changes. The PR title should be the first thing displayed when the PR is opened. And it should follow the semantic commit rules. For example, a title like "docs(#issueId): Add README.md" can be used.
- [x] Related file selection: Only relevant files should be touched and no other files should be affected.
- [x] Format and Lint Suitability : The code should be made in accordance with a certain format standard and examined according to the lint rules.
- [x] Clean commit history: The commits where the changes are made should be clear and organized.
- [x] Opening PR when job is completed: PR should be opened when job is completed and sent for review by other team members.

## Changes

- [x] A new feature (a change that adds a new feature, not a breaking change)
- [ ] A new refactor (a change that is not a breaking change, that improves the readability or performance of the code)
- [ ] A breaking change
- [ ] Documentation change

# How were these changes tested?

Please describe the tests you did to test the changes you made. Please also specify your test configuration.

**Test Configuration**:

* Firmware version:
* Hardware:
* Toolchain:
* SDK:

### CODE REVIEW

**Review by oguzergul — APPROVED** [review]

**Comment by birkaany** [comment]
Closing this pr until the reference issue is approved.

**Review by oguzergul — APPROVED** [review]

**Review by cdagli — APPROVED** [review]

**Comment by usirin** [comment]
@oguzergul @cdagli this pr introduced a build error problem: we added ssg with this PR, whereas it was already using ssr with getInitialProps.

i sent #766 to unblock the pipeline but @oguzergul can you please create an issue for this to get to the bottom of this?

**Pull request merged**
